### PR TITLE
Updated keyboard shortcut

### PR DESCRIPTION
**Pull request by erdraud**

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Command Palette provides access to Microsoft Edge commands, including various De
 
 To open Command Palette:
 
-1. In Microsoft Edge, press `Ctrl`+`Shift`+`Space`.  Command Palette opens.
+1. In Microsoft Edge, press `Ctrl`+`Q`.  Command Palette opens.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:44]: Stable 105 Win: Ctrl+Shift+Space works.  Ctrl+Q does nothing.
Canary v__ Win: tbd

Stable 105 Mac: tbd
Canary v__ Mac: tbd

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:44]: The keyboard shortcut change is happening right now. Maybe you just haven't got the new code yet? @erdraud when will the new shortcut be available to test in Edge?

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:44]: @captainbrosset - Per erdraud, the Ctrl+Q shortcut key will be in Canary the week of September 19, 2022. The experiment (in flags page) is the old shortcut. The rollout (not flags page) is Ctrl+Q.

Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:44]:
```suggestion
1. In Microsoft Edge 108 or later, press `Ctrl`+`Q`.  Or, in Microsoft Edge 107 or earlier, press `Ctrl`+`Shift`+`Space`. Command Palette opens.
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Command Palette provides access to Microsoft Edge commands, including various De
 
 To open Command Palette:
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:43]: Add a statement regarding the screenshot.  It currently shows old shortcut, not new Ctrl+Q shortcut.  So tagging could be:

```

The above screenshot shows the shortcut for Microsoft Edge 105-107.  Microsoft Edge 108 or later will show the shortcut `Ctrl+Q` instead.
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md (hunk 1 above)
Comment (erdraud) [reply microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:44]: @captainbrosset we have a PR on standby

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md]: @captainbrosset - Per erdraud, The Ctrl+Q shortcut will be in by 108, possibly in as early as a later minor release of 107, but not yet in current minor release of 107.  2 changes: The displayed text in the flags page, and the behavior.

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md (hunk 2 above)
Comment (erdraud) [reply microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:44]: Added this line

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md [hunk 3]
```diff
@@ -41,7 +42,7 @@ Command Palette provides access to Microsoft Edge commands, including various De
...
 
-1. In Microsoft Edge, press `Ctrl`+`Shift`+`Space`.  Command Palette opens.
+1. In Microsoft Edge 108 or later, press `Ctrl`+`Q`.  Or, in Microsoft Edge 105-107, press `Ctrl`+`Shift`+`Space`. Command Palette opens.
 
 1. Start typing in the input box. For example:
    * Type **tabs** to display commands about tabs management.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:48]: In 3 places, change from:
```
In Microsoft Edge, press Ctrl+Shift+Space. Command Palette opens.
```
to:
```
In Microsoft Edge 108 or later, press `Ctrl`+`Q`. Or, in Microsoft Edge 105-107, press `Ctrl`+`Shift`+`Space`. Command Palette opens.
```

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md:48]: Confirmed fixed.  ok to Resolve this Conversation.

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**